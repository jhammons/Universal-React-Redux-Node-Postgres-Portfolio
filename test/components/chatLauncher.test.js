import React from 'react';
import { shallow } from 'enzyme';

describe('<ChatLauncher />', () => {
    it('it renders', () => {
        expect(true).toEqual(true);
    });
});
