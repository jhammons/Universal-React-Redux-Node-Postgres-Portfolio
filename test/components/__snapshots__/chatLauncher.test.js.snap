// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LiveChatLauncher /> it matches snapshot 1`] = `
<div
  className="chat-launcher"
  onClick={[Function]}
>
  <div
    className="chat-indicator chat-offline"
  />
  Live
  <br />
  Chat
</div>
`;
